commands.py

```diff
@@ -62,7 +85,10 @@ def color(update, context):
 
     if not someone_worked:
         context.bot.send_message(
-            chat_id=update.effective_chat.id, text="Perdão, deu erro nas minhas dependências |:(")
+            chat_id=update.effective_chat.id,
+            text=dependencies_error_message.get_text(
+                update.message.from_user.language_code[:2])
```

Comment:
Deveras astuto pegar a língua do telegram da pessoa

Reply:
Thank you ✋